### ISSUE TITLE
Unit test hangs because Sinon stub doesn't work

### ISSUE THREAD

**Issue by jaekunchoi** [jeroenouw/cgx#12]
**Describe the bug**
When I run `npm run test:unit` the test hangs with Inquirer prompt and Sinon stub doesn't get used

**To Reproduce**
1.  add `.mocharc.js` with below code in it to run with new Mocha version and node v14.14.0

```
'use strict';

module.exports = {
  diff: true,
  extension: ['ts'],
  reporter: 'spec',
  recursive: true,
  require: ['ts-node/register', 'test/mocha.require.ts'],
  spec: 'src/**/*.spec.ts',
  watch: true
}
```
2. Run `npm run test:unit`

**Expected behavior**
Test should run fine

**Screenshots**
<img width="810" alt="Screen Shot 2020-10-19 at 11 22 59 pm" src="https://user-images.githubusercontent.com/1248928/96450516-0c74ac80-1262-11eb-912c-0eb22660e1b0.png">

**Comment by jeroenouw** [comment]
@farisT can you take a look at this?

**Comment by farisT** [comment]
@jeroenouw will pick this up

**Comment by jaekunchoi** [comment]
@farisT how is it going? Do you need more context on this?

**Comment by farisT** [comment]
@jaekunchoi still looking at why it does sinon does not stub anymore (seems to be stubbing other functions fine) - we can try using an alternative like https://github.com/thlorenz/proxyquire. Unless you can think of maybe other alternatives?

**Comment by farisT** [comment]
The issue is because of the transpiling of the modules using import/export. Refer to this thread https://github.com/sinonjs/sinon/issues/1711. Therefore when you refer to the `index.ts` it cannot find what to stub. The solution is either referring to the module directly in the stubbed tests eg.
`
import * as actions from './actions/github.actions';
`
Or using something like proxyquire as I mentioned previously. What do you think @jeroenouw ?

**Comment by jeroenouw** [comment]
@farisT referring directly to the module is ok.

**Comment by jeroenouw** [comment]
@jaekunchoi should be solved in [#13 ](https://github.com/jeroenouw/cgx/pull/13) by @farisT
